README.md

```diff
@@ -44,6 +44,7 @@ Call the `dirtyCheck` function, which accepts three arguments:
 3. Config:  
   - `debounce` - debounce time of `valueChanges`. Defaults to 300  
   - `withDisabled` - whether to include disable fields (by using control's `getRawValue`) or not. Defaults to `true`. 
+  - `window` - global `Window` object to handle `onbeforeunload` event. Defaults to `undefined`.
```

Comment:
This is a breaking change

Reply:
Yes, I know. Is it impossible to mark current behavior as deprecated?